tests/faultydata/variants/duplicated_sampleid/README.md

```diff
@@ -0,0 +1,6 @@
+Data file example_1.vcf.gz was copied from tests/data/variants/example_1, then change sample id S2 to S1
+
+Index file was generated by
+```
```

Comment:
I don't think we should assume one's editor has markdown support.  Mine doesn't, and every text editor supports plaintext!

But I want to hear @jeromekelleher 's thoughts on this, although my inclination is towards .txt files.